apps/rewards/package.json

```diff
@@ -31,7 +31,7 @@
   },
   "dependencies": {
     "@aragon/api": "2.0.0-beta.8",
-    "@aragon/api-react": "2.0.0-beta.6",
+    "@aragon/api-react": "2.0.0-beta.8",
```

Comment:
It is needed to update the rest of apps as well, otherwise the custom api-react logic from the stubbed api will use the hoisted version (the old one in this case), causing error in the frontend and preventing the app to launch.

Reply:
Fixed in 5b4e86af98d96ab1aee064fd9341a66d7f03276f